modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -110,7 +110,8 @@
 			var/reagent = GLOB.name2reagent[reagent_name]
 			if(!(reagent in cortical_owner.known_chemicals))
 				return
-
+			if(isnull(cortical_owner.reagent_holder)) // BUBBER EDIT FIX
+				cortical_owner.reagent_holder = new /obj/item/reagent_containers/borer(src) // BUBBER EDIT FIX
```

Comment:
```suggestion
			if(isnull(cortical_owner.reagent_holder)) // BUBBER EDIT FIX
				cortical_owner.reagent_holder = new /obj/item/reagent_containers/borer(cortical_owner) // BUBBER EDIT FIX
```

Whilst this *probably* won't be an issue the container should be created within the same parent